### DISCUSSION TITLE
Can't query basic data about my Vault

### DISCUSSION THREAD

**Discussion by aagha** [brianpetro/obsidian-smart-connections#503] [Q&A]
I'm not understanding why, after 1100 notes have been indexed, why are questions like this being answered incorrectly? 

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/5522103/81141bcf-0c84-4b1c-bb34-75fee0e91307)

In addition, there's major errors in what it's finding when I query it:

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/5522103/2c968615-d940-4234-989c-a5106aca672b)

That's wrong...

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/5522103/6a015e34-f3ca-4924-a405-d67971137e7d)

Have I setup something incorrectly?

**Comment by brianpetro** [comment]
Only the top 20 notes can fit into the context (unless you use GPT-4-turbo with 128K context.

And the way semantic (embedding) search works doesn't allow for many traditional keyword queries, which would be required for the "Drew" example. 

🌴

**Comment by aagha** [comment]
Got it. Thanks @brianpetro.
